packages/jsii-dotnet-runtime/src/Amazon.JSII.Runtime/Services/NodeProcess.cs

```diff
@@ -9,25 +9,36 @@ public class NodeProcess : INodeProcess
     {
         readonly Process _process;
         readonly ILogger _logger;
+        private const string JsiiRuntime = "JSII_RUNTIME";
+        private const string JsiiDebug = "JSII_DEBUG";
+        private const string JsiiAgent = "JSII_AGENT";
 
         public NodeProcess(IJsiiRuntimeProvider jsiiRuntimeProvider, ILoggerFactory loggerFactory)
         {
             loggerFactory = loggerFactory ?? throw new ArgumentNullException(nameof(loggerFactory));
             _logger = loggerFactory.CreateLogger<NodeProcess>();
 
+            var runtimePath = Environment.GetEnvironmentVariable(JsiiRuntime);
+            if (string.IsNullOrWhiteSpace(runtimePath))
+                runtimePath = jsiiRuntimeProvider.JsiiRuntimePath;
+            
             _process = new Process
             {
                 StartInfo = new ProcessStartInfo
                 {
                     FileName = "node",
-                    Arguments = "--max-old-space-size=4096 " + jsiiRuntimeProvider.JsiiRuntimePath,
+                    Arguments = "--max-old-space-size=4096 " + runtimePath,
                     RedirectStandardInput = true,
                     RedirectStandardOutput = true,
                     RedirectStandardError = true
                 }
             };
 
-            _process.StartInfo.EnvironmentVariables.Add("JSII_AGENT", "DotNet/" + Environment.Version);
+            _process.StartInfo.EnvironmentVariables.Add(JsiiAgent, "DotNet/" + Environment.Version);
```

Comment:
Any chance we can be more "accurate" than "DotNet"? Is it possible to get ideas of the framework build version or something?

Reply:
Yup, we do it in the SDK, will do the same here.